MapboxCoreNavigation/RouteController.swift

```diff
@@ -88,7 +88,11 @@ open class RouteController: NSObject {
     /**
      The route controller’s associated location manager.
      */
-    public var locationManager: NavigationLocationManager!
+    public var locationManager: NavigationLocationManager! {
+        didSet {
+            locationManager.delegate = self
```

Comment:
Good idea. Should we unset `oldValue`’s delegate too, just in case the developer swaps location managers?